packages/example/android/app/build.gradle

```diff
@@ -105,6 +114,9 @@ repositories {
   maven {
     url "${KLARNA_DISTRIBUTION_URL}"
   }
+    maven { // TODO TWS: remove before release
+        url "https://s01.oss.sonatype.org/content/repositories/staging/"
```

Comment:
Is there a separate ticket/task for this todo?

Reply:
There's a release prep ticket that would cover this, but I'll get rid of this repo now, as we're already using the production repo for the Stripe wrapper.